src/client/entity-editor/name-section/actions.ts

```diff
@@ -26,7 +26,7 @@ export const UPDATE_NAME_FIELD = 'UPDATE_NAME_FIELD';
 export const UPDATE_SORT_NAME_FIELD = 'UPDATE_SORT_NAME_FIELD';
 export const UPDATE_WARN_IF_EXISTS = 'UPDATE_WARN_IF_EXISTS';
 export const UPDATE_SEARCH_RESULTS = 'UPDATE_SEARCH_RESULTS';
-
+export const UPDATE_COPY_CHECKBOX = 'UPDATE_COPY_CHECKBOX';
```

Comment:
Same remark as my other comment, let's rename this `UPDATE_COPY_NAME_CHECKBOX` or `UPDATE_COPY_NAME_TO_EDITION_GROUP` to make way for `UPDATE_COPY_AUTHOR_CREDIT_…`